_hacks/10.md

```diff
@@ -0,0 +1,14 @@
+---
+hack_number: 10
+date: 2021-01-30
+happened: true
+---
+We had our most _international_ day of remote hacks yet, with participants California and Munich!
+
+### Things people did
+
+- **Ben** pulled **everyone** down the rabbit hole of LIDAR/PCAP and also some AWS hacks.
+- **Max** intended to work on his LensLok solver, but instead worked on the Airquotes the Podcast Airquotes _branding_
+- **Panda** worked on some software to help keep his plants alive.
+- **Dan** learned some Typescript.
+- **Ruth** etched a design and cut out a jigsaw using her laser cutter, and also hacked a bit on some WebAudio convolution.
```

Comment:
Ah, I plonked down some notes the other day.

I've mashed them in (might need some tweaking or more info):

```suggestion
- **Henry** joined us from California, where it was 2am! Simultaneously becoming our remote-ist and our most nocturnal hacker.
- **Ruth** showed people her laser cutter, which she used to etch & cut a jigsaw 🧩⚡️. Then she did some hacking with convolution filters using WebAudio 🎷.
- **Dimitri** was planning to _learn_ **Rust** by using it to compile **Elm** to **JavaScript**.  Though he ended up helping people find bugs before they'd even written the code that introduced them 💅.
- **Panda** planned to write some software to help keep his plants alive, but instead got sucked into Bens AWS vortex (🌱💀)
- **Max** intended to work on his LensLok solver, but instead worked on the Airquotes the Podcast Airquotes _branding_
- **Mike** used NextJS and MDX to build a blog with dynamic graphs in it!
- **Dan** learned some Typescript, then taught people aws, dns & service features, and then became an elite streamer by mastering OBS
- **Ben** pulled **Dan**, **Dimitri**, **Ryan**, **Panda** and **everyone** down the rabbit hole of LIDAR/PCAP and AWS hacks. (he's very sorry, but also had a great time.)
- **Alan** got to grips with static site building with 11ty 👨‍💻
- **Emanuil** joined us in the afternoon, and worked on a FOSSJobs site 👍 
- **Piers** told us about his VW Camper hack and chatted about augmented reality, then programmed a microcontroller to measure distances.
- **Adam** intended to create some statically rendered flashcards with nextJS but ended up just interviewing people and getting a PR Merged into Ruth's generative artistry website
```

Reply:
💪

Reply:
I'll take your changes (need to commit it manually 😓 )

Reply:
Resolved in 0880a2f